lib/assets.js

```diff
@@ -12,14 +12,14 @@ var defaults = {
   }
 };
 
-module.exports = function (gulp, options) {
-  var opts = merge({}, defaults, options);
-  var dest = opts.dest;
-  var optimize = opts.optimize;
-  var plugins = opts.plugins;
-  var src = opts.src;
+module.exports = (gulp, options) => {
+  const opts = merge({}, defaults, options);
```

Comment:
Excellent, thanks so much @erikjung! That article was _very_ helpful in better understanding `let`. :)